frontend/src/component/Search/AdvancedSearchContentList.jsx

```diff
@@ -26,6 +26,12 @@ export const AdvancedSearchContentList = props => {
     }
   })
 
+  const numberCommentsTitle = (numberComments) => {
+    if (numberComments === 0) return props.t('0 comments')
+    if (numberComments === 1) return props.t('{{numberComments}} comment', { numberComments: numberComments })
+    else return props.t('{{numberComments}} comments', { numberComments: numberComments })
+  }
+
```

Comment:
i18next can [manage plurals](https://www.i18next.com/translation-function/plurals), any reason you didn't use it here?